service/package/gem2rpm.yml

```diff
@@ -26,6 +26,7 @@
     Provides:       agama-yast
     BuildRequires:  dbus-1-common
     Requires:       dbus-1-common
+    Requires:       dbus-1-daemon
```

Comment:
do we need it also for rust dbus server or deps live only here?

Reply:
I decided to put it only there because the rust package depends on agama-yast already.